pydarn/utils/conversions.py

```diff
@@ -81,7 +81,26 @@ def dmap2dict(dmap_records: List[dict]) -> List[dict]:
         dmap_list.append(OrderedDict(dmap_dict))
     return dmap_list
 
+def gateGroundScatter(y, Re, hgt=250):
+    """
+    Calculate the ground scatter mapped range (km) for each range gate for SuperDARN data
```

Comment:
If there is a  published paper this work is based on then I would put it in for people's further understanding.

Reply:
Oo also amend to gate2GroundScatter given above comments

Reply:
Sorry about that I added it as a comment I have now moved it into the the doc string